ovirtlago/virt.py

```diff
@@ -262,8 +264,28 @@ def _artifact_paths(self):
         inherited_artifacts = super(EngineVM, self)._artifact_paths()
         return set(inherited_artifacts + ['/var/log'])
 
+    def _download_ca_cert(self):
+        url = constants.ENGINE_BASEURL.format(
+            self.ip()
+        ) + '/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA'
+        retries = 20
+        while retries:
+            resp = requests.request('GET', url, verify=False)
+            if resp.ok:
+                with open('engine-ca.pem', 'w+b') as outfile:
+                    outfile.write(resp.content)
+                self._cert = 'engine-ca.pem'
```

Comment:
I think that we should add a new value called `engine_cert` to `paths` module, and use it for storing the certificate.
Using this method the certificate will be saved inside the prefix, and the path to it will be absolute.